### DISCUSSION TITLE
7.1.22

### DISCUSSION THREAD

**Discussion by blairmcg** [dolphinsmalltalk/Dolphin#1161] [Releases]
Replaces obsolete Internet Explorer integration with Edge (Chromium) based WebView2

## What's Changed
* Add WebView2 package (see #1156) by @blairmcg in https://github.com/dolphinsmalltalk/Dolphin/pull/1157
* Improve WebView2 abstraction and replace all uses of IE with it by @blairmcg in https://github.com/dolphinsmalltalk/Dolphin/pull/1158
* Resave URLPresenter WebView2 view for new inst var layout by @blairmcg in https://github.com/dolphinsmalltalk/Dolphin/pull/1159
* Some tidy up of WebView2 by @blairmcg in https://github.com/dolphinsmalltalk/Dolphin/pull/1160

<hr /><em>This discussion was created from the release <a href='https://github.com/dolphinsmalltalk/Dolphin/releases/tag/7.1.22'>7.1.22</a>.</em>

**Comment by JBetz** [comment]
Awesome!
